src/icons/states/LocationErrorState.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { boxBlue, white } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    opacity?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const LocationErrorState = ({
+    className = '',
+    color = boxBlue,
+    height = 111,
+    opacity = 0.1,
+    title,
+    width = 130,
+}: Props) => (
+    <AccessibleSVG
+        className={`LocationErrorState ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 140 111"
+        width={width}
+    >
+        <g fill="none" fillRule="evenodd">
+            <path
+                fill={color}
```

Comment:
Check the other state icons, and if they are doing it, add classes like fill-color or stroke-color (yeah old style for now) to each path entry for custom branding override. Ignore if the state should not have custom branding override. I think we just either use black or blue?

Reply:
We use black for custom branding. I don't see the classes in other state icon and the color seems to be coming just from the prop itself.

Reply:
yeah but you will see things like

className="stroke-color"
className="fill-color"

in a few icons which is what gets overridden for custom branding